simplebgc/src/commands/motors_off.rs

```diff
@@ -9,3 +12,31 @@ pub enum MotorsOffMode {
     /// off completely
     SafeStop,
 }
+
+#[derive(BgcPayload, Copy, Clone, Debug, PartialEq)]
+pub struct MotorsOffQuery {
+    #[kind(enumeration)]
+    #[name("")]
+    #[format(u8)]
+    pub mode: MotorsOffMode,
+}
```

Comment:
If this is an unnamed property, shouldn't we just make this a tuple struct?

Reply:
done